data/map planets.txt

```diff
@@ -4958,3 +4958,67 @@ planet Zydee
+	landscape land/lava3
+#	description "Bailey is a dead, nearly tidally locked, world scarcely larger or more habitable than Mercury. Its cool side has hardened into a mineral-rich crust, a rarity in this region of space. Though miners must regularly be on the move it has brought several mining expeditions to the planet. Like aphids sucking the sap from plants, semi-mobile mining stations strip and refine the minerals so that they may be used by the CCOR."
+	spaceport "Given the roving nature of the settlements, and the lack of general atmosphere, the spaceport and other major infrastructure are located in orbit. Transit to and from the planet is facilitated by a clever network of skyhooks, which utilize smaller amounts of rare and expensive resources compared to the more common repulsor technology used elsewhere."
+	bribe 1
+	security 0.33
+
+planet "M.B. Station"
+	attributes CCOR military station urban
+	landscape land/city6
+	description "The hub of military logistics for the CCOR Mordente-Bridi station is as close to a fortress as can be built without the benefit of an asteroid field. The twin cylinders that make up the outer hull are layered with the strongest armor CCOR can manufacture at scale. The Bridi Cylinder is pressurized containing crew quarters and logistical facilities, and the Mordente contains CCOR's shipyard."
+	spaceport "Almost none of the Mordente Cylinder is pressurized. Within the armored hull is a micro-gravity shipyard complete with worker-shuttles busily zipping about constructing and repairing a plethora of ships and the fighter hangars in addition to the spaceport."
+	shipyard "Advanced Southern Pirates"
+	shipyard "Betelgeuse Advanced"
+	shipyard "Betelgeuse Basics"
+	shipyard "Southern Pirates"
+	outfitter "Ammo South"
+	outfitter "Basic Outfits"
+	outfitter "Basic Weapons"
+	outfitter "Common Outfits"
+	outfitter "Pirate Outfits"
+	security 0.5
+
+planet "Mendez Station"
+	attributes CCOR station
+	landscape land/station20
+#	description "Mendez Station is the closest object to Fumalasamakah. Located in a low orbit just above the stellar corona, the monumentous ram-scoops built into the star-facing side of the station collect stellar ejections to process into fuel. The station is perpetually hot, even in the habitation sections as an inescapable effect of begin so close to a great mass of plasma undergoing nuclear fusion, however it would surely be much more so if it weren't for the primarily polyfiber construction and heat-radiation spires built into the far side of the station."
+#	spaceport "Rather than a traditional docking ring, part of which would be facing Fumalasamakah and its incredible heat and radiation at all times Mendez's 'spaceport' is actually a separate station located in a significantly higher stellar orbit. The station serves primarily as a secure dock for more normal ships while their inhabitants are taken to Mendez station via specialized transports built to resist the intense heat."
```

Comment:
```suggestion
#	spaceport "Rather than a traditional docking ring, part of which would have been facing Fumalasamakah and its incredible heat and radiation at all times, Mendez's 'spaceport' is actually a separate station located in a significantly higher stellar orbit. The station serves primarily as a secure harbor for normal ships while their occupants are taken to Mendez station via specialized transports built to resist the intense heat."
```